src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -2004,4 +2004,13 @@ Additional information: {1}</value>
   <data name="Document_Outline" xml:space="preserve">
     <value>Document Outline</value>
   </data>
+  <data name="Show_ui_as_dashboard_in_top_right" xml:space="preserve">
+    <value>Show UI as a dashboard in top right</value>
```

Comment:
ah, you are using UI in the actual user string.  then can you update hte loc key to match that?  (note this is a benefit of keeping both in sync. it means at teh code point it's clear what the user will see.